starter/script.js

```diff
@@ -1 +1,21 @@
+// Kory J. Campbell Choose Your Own Adventure Game
+
+alert("Welcome to Joe's Pizzeria Delivery! A Text-Based Adventure. Press OK.");
+alert("You are a delivery bicycler who has to deliver pizzas to your customers safely! What will this entail..... Press Ok.");
+
+var userInput = prompt("You walk out the Pizzeria, on your way to your customer and you see a little red-haired boy who ask for a slize of pizza. Do you want to give him a slice of your pizza? [type Y or N]");
+
+if(userInput == "Y") {
+  prompt("The red-haired boy takes your pizza happily...however after he quickly gulps it down he asks for another slice...do you fulfill his greedy wishes? [type Y or N]");
+}
+
+if(userInput == "Y") {
+  alert("The red-haired boy eats your slices...and as you lay the pizxa down in preparation to get on your bike, he steals your entire pizza. :( GAMEOVER");
+} else(userInput == "N"); {
```

Comment:
An `else` block contains code that runs if the conditions outlined in your `if` and `else if` blocks aren't triggered. It is never paired with a specific condition. So it usually looks something like this...

```js
if ( x === 2 ){
  console.log( "x is 2!" );
} else if ( x === 3 ) {
  console.log( "x is 3!" );
} else {
  console.log( "x is something else!" );
}
```